roles/lib_utils/filter_plugins/oo_filters.py

```diff
@@ -660,6 +660,47 @@ def map_from_pairs(source, delim="="):
     return dict(item.split(delim) for item in source.split(","))
 
 
+def lib_utils_oo_get_node_labels(source, hostvars=None):
+    ''' Return a list of labels assigned to schedulable nodes '''
+    labels = list()
+
+    # Filter out the unschedulable nodes
+    for host in source:
+        if host not in hostvars:
+            return
+        node_vars = hostvars[host]
+
+        # All nodes are considered schedulable,
+        # unless explicitly marked so
+        try:
+            if not strtobool(str(node_vars.get('openshift_schedulable'))):
+                # explicitly marked as unschedulable
+                continue
+        except ValueError:
```

Comment:
None type will raise this exception.  If None type, we want to add the labels.

Reply:
Fixed, now converting `None` to `True` there